src/main/java/entity/User.java

```diff
@@ -6,19 +6,16 @@
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
 
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
 
 public class User {
 
     @NotEmpty
     private String id;
-    private List<@Valid User> followingList;
-    private List<@Valid User> followerList;
-    private List<@Valid Post> postList;
-    private List<@Valid Post> likedPostList;
+    private List<String> following;
+    private List<String> followers;
```

Comment:
```suggestion
    private List<String> followerIdList;
```
Add id to variable name for clarity of what list stores